src/components/modals/HelpModal.tsx

```diff
@@ -162,8 +163,8 @@ const getStyles = cacheStyles((theme: Theme) => ({
     height: theme.rem(2.25)
   },
   footer: {
-    marginTop: PLATFORM.deviceHeight < theme.rem(42) ? 0 : theme.rem(1.5),
-    paddingVertical: PLATFORM.deviceHeight < theme.rem(42) ? theme.rem(0.25) : theme.rem(0.5),
+    marginTop: deviceHeight < theme.rem(42) ? 0 : theme.rem(1.5),
+    paddingVertical: deviceHeight < theme.rem(42) ? theme.rem(0.25) : theme.rem(0.5),
```

Comment:
Might be an opportunity to explain this logic.

Reply:
I have no idea.

Reply:
Ha! Okay